sdk/python/kfp/dsl/_component_bridge.py

```diff
@@ -46,20 +46,36 @@ def _create_container_op_from_component_and_arguments(
 
     old_warn_value = dsl.ContainerOp._DISABLE_REUSABLE_COMPONENT_WARNING
     dsl.ContainerOp._DISABLE_REUSABLE_COMPONENT_WARNING = True
+
+    # Merge output_paths and output_uris to get the file_outputs.
+    file_outputs = resolved_cmd.output_paths or {}
+    for name, output_uri in resolved_cmd.output_uris.items():
+        file_outputs[name] = output_uri
+
+    artifact_argument_paths = [
+        dsl.InputArgumentPath(
+            argument=arguments[input_name],
+            input=input_name,
+            path=path,
+        )
+        for input_name, path in resolved_cmd.input_paths.items()
+    ]
+
+    for input_name, input_uri in resolved_cmd.input_uris.items():
```

Comment:
Yes it can, but I'm slightly leaning towards keep as-is for readability, and it's seems not very trivial to inline the dict merging without making the list comprehension too complex.